src/insexpand.c

```diff
@@ -4456,7 +4457,7 @@ ins_compl_expand_multiple(char_u *str)
 		ins_char_bytes(start, (int)(curr - start));
 
 	    // Handle newline
-	    open_line(FORWARD, OPENLINE_KEEPTRAIL, FALSE, NULL);
+	    open_line(FORWARD, OPENLINE_KEEPTRAIL | OPENLINE_FORCE_INDENT, base_indent, NULL);
```

Comment:
Why doesn't `ins_compl_expand_multiple` temporarily set 'paste'? It should not be subject to any insert-mode behaviors while inserting the text.

Reply:
I tried but it didn't work. Maybe there is something wrong. I will try again.